meta/main.yml

```diff
@@ -5,7 +5,7 @@ galaxy_info:
   company: pro!vision
   issue_tracker_url: https://wcm-io.atlassian.net
   license: Apache
-  min_ansible_version: 2.0
+  min_ansible_version: 2.5
```

Comment:
I am wondering that this is working together with travis, since travis is using 2.0 for the CI. But the CI only performs a syntax check an a lint.
We should increase the version in .travis.yml too, see https://github.com/wcm-io-devops/ansible-conga-maven/blob/master/.travis.yml#L9

Reply:
Thanks, good point